cmd/monaco/test-resources/integration-all-configs/project/failure-detection-parametersets/parameterset.json

```diff
@@ -0,0 +1,20 @@
+{
+    "id": "{{ .id }}",
```

Comment:
Please remove the id from the json. `monaco` resolves all configs by name.

Reply:
This is a little exotic but the description of the API says the following: 

> The JSON body of the request. Contains the new failure detection parameter set.
> 
> Dynatrace does not generate the ID automatically; you must specify it.

I hope this is no problem.

Reply:
Sadly at the moment the ID is mandatory, so it will not work. The API will be changed is the latest information I got. I will do the changes when the API is updated.

Reply:
:cry: Ok, that's unfortunate for monaco. monaco works by resolving all configs by name. So it relies on the auto-generation of the id by the API. So, we need to wait with this PR until the API is fixed.

Reply:
We will do and will solve this when the API is fixed. Code as it is pushed can be used anyways (with the changes done you mentioned)

Reply:
The PR is brought back to life as the fixed API is now available 🤞🏻